server.js

```diff
@@ -24,9 +17,116 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
+const Animal = mongoose.model("Animal", {
+  name: String,
+  scientificName: String,
+  doesItFart: Boolean,
+  doesItMaybeFart: Boolean,
+  descriptionExists: Boolean,
+  descriptionNotes: String,
+  paperLink: String,
+  verifiedBy: String,
+});
+
+// if (process.env.RESET_DB) {
+// const seedDatabase = async () => {
+//   await Animal.deleteMany();
+//   doItBeFartin.forEach((animal) => {
+//     new Animal(animal).save();
+//   });
+// };
+// seedDatabase();
+// }
+
+app.use((req, res, next) => {
+  if (mongoose.connection.readyState === 1) {
+    next();
+  } else {
+    res.status(503).json({ error: "Service unavailable" });
+  }
+});
```

Comment:
Very "handy" block of code  - an accessibility improvement in order for the viewer to get this error message if mongoose doesn't connect. Awesome!